.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ env:
 
 jobs:
   build-linux:
-    runs-on: ubuntu-20.04 # hopefully older glibc for better compatibility on systems like debian bullseye
+    runs-on: ubuntu-22.04
```

Comment:
undo this

Reply:
The action will not run per https://github.com/actions/runner-images/issues/11101 and from my testing

Reply:
I see. But we have to verify that this update is okay. Like the comment says we want an older glibc as glibc isn't forwards compatible

Users with an older glibc than the one used to compile won't be able to run it